bin/start_tests.sh

```diff
@@ -15,6 +15,9 @@ cd cli
 cp tests/data/dcos.toml $DCOS_CONFIG
 sed -i "s/change.dcos.url/$VBOX_IP/g" $DCOS_CONFIG
 
+# overwrite core.dcos_url in the ssl.toml
+sed -i "s/change.dcos.url/$VBOX_IP/g" tests/data/ssl/ssl.toml
```

Comment:
This means that running the test would change the repository. This is the reason why we `cp` the file in line 15 above.

I think you won't need to do this if you allow environment variable to take precedence over `dcos.toml`. Thoughts?